src/Confluent.Kafka/Consumer.cs

```diff
@@ -858,227 +172,75 @@ public Consumer(IEnumerable<KeyValuePair<string, object>> config)
             }
         }
 
-        /// <summary>
-        ///     Raised on new partition assignment.
-        ///     You should typically call the Consumer.Assign method in this handler.
-        /// </summary>
-        /// <remarks>
-        ///     Executes on the same thread as every other Consumer event handler (except OnLog which may be called from an arbitrary thread).
-        /// </remarks>
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="OnPartitionsAssigned"]/*' />
```

Comment:
unfortunately, docfx (the tool we're using to create API documentation) doesn't understand these include directives, so API documentation is no longer being created for these methods. There is too much duplication now to consider copy/pasting all this, so I propose that for now the auto-generated API documentation is going to have to suffer. I think we will probably need to either enhance docfx, or write a preprocessing tool to replace the include directives with the docs they reference. This only applies to the generated HTML documentation - these comments also form the basis of the intellisense text (everything is good here).